tf2rl/experiments/trainer.py

```diff
@@ -27,6 +27,13 @@ def __init__(
             env,
             args,
             test_env=None):
+        if isinstance(args, dict):
+            _args = args
+            args = policy.__class__.get_argument(Trainer.get_argument())
+            args = args.parse_args([])
+            for k, v in _args.items():
+                setattr(args, k, v)
```

Comment:
I guess this part overrides the arguments by specified dictionary.
Would it be possible to reject elements that are not defined in the `args = policy.__class__.get_argument(Trainer.get_argument())`?

Reply:
As long as `argparse` guarantees that there exist all the possible parameters including parameters not specified by command line, we can check with `hasattr(args, k)`.

Reply:
@keiohta 

I added a commit (a343fff) which makes `Trainer` raise `ValueError` against unsupported arguments.

Reply:
LGTM!